src/data/charts.json

```diff
@@ -340,6 +341,7 @@
             "label": "",
             "x": "numberOfLandOwnersPerNationality",
             "y": "total",
+            "aggregate": "percent",
```

Comment:
Out of scope (may be) but we could chain these aggregate function definitions me thinks. Example `sum-percent` would clearly indicate we need to sum first. `percent` by itself means just that, show this data as percentage.

Reply:
`sum:percent` splendid suggestion @kilemensi